### PR TITLE
Feature/add kaminari-gem

### PR DESCRIPTION
**Pull request by jesusMava**
Add kaminary gem and delete haml and unnecessary controller Main.

### CODE REVIEW

File: app/controllers/curiosity_cards_controller.rb [hunk 1]
```diff
@@ -4,10 +4,21 @@ class CuriosityCardsController < ApplicationController
   before_action :find_curiosity_scope_by_current_user, only: [:update, :destroy]
 
   def index
-    @curiosities = CuriosityCard.all
+    @curiosities = CuriosityCard.order('created_at DESC').page(params[:page]).per(12)
```
Comment (itsmeurbi) [inline app/controllers/curiosity_cards_controller.rb:7]: You could use `.order(created_at: :desc)` instead

**Review by itsmeurbi — CHANGES_REQUESTED** [review]
Ok first thing first. 
PR name says "add kaminari and instant search" but I see you included a pdf export functionality.
Can we split this PR, separating features. We should have 1 PR implementing pagination and another one adding an export functionality

File: app/controllers/curiosity_cards_controller.rb (hunk 1 above)
Comment (jesusMava) [reply app/controllers/curiosity_cards_controller.rb:7]: this `.order(create_at: :desc) `  generate a error Dangerous query method (method whose arguments are used as raw SQL) called with non-attribute argument(s): "created_at: :desc".This method should not be called with user-provided values, such as request parameters or model attributes. Known-safe values can be passed by wrapping them in Arel.sql().

File: app/controllers/curiosity_cards_controller.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@ class CuriosityCardsController < ApplicationController
   before_action :find_curiosity_scope_by_current_user, only: [:update, :destroy]
 
   def index
-    @curiosities = CuriosityCard.all
+    @curiosities = CuriosityCard.order_curiosity_by_desc.page(params[:page]).per(12)
```
Comment (itsmeurbi) [inline app/controllers/curiosity_cards_controller.rb:7]: Model name is `CuriosityCard`. That being said, `order_curiosity_by_desc` sound redundant and not so clear 
Maybe just `ordered_by_recents`? 🤔

File: app/models/curiosity_card.rb [hunk 3]
```diff
@@ -13,4 +13,6 @@ class CuriosityCard < ApplicationRecord
   has_many :users, through: :comments
 
   enum action: { fake: 0, real: 1 }
+
+  scope :order_curiosity_by_desc, -> { order('created_at Desc') }
```
Comment (itsmeurbi) [inline app/models/curiosity_card.rb:17]: I'd use sql statements in capital(i.e. DESC)

**Review by itsmeurbi — CHANGES_REQUESTED** [review]
PR name is `Feature/add kaminari-gem` but you're getting rid of main controller and haml?

**Pull request closed without merging**